timesketch/api/v1/resources.py

```diff
@@ -338,12 +373,66 @@ def post(self, sketch_id):
         form = SaveViewForm.build(request)
         if form.validate_on_submit():
             sketch = Sketch.query.get_with_acl(sketch_id)
+            new_searchtemplate = form.new_searchtemplate.data
```

Comment:
Done. I'm not 100% sure this is optimal for readability. I'm gonna revisit this in a later CL.